src/ResourceManager/Compute/Commands.Compute/help/Export-AzureRmLogAnalyticRequestRateByInterval.md

```diff
@@ -0,0 +1,233 @@
+---
+external help file: Microsoft.Azure.Commands.Compute.dll-Help.xml
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.compute/export-azurermloganalyticrequestratebyinterval
+schema: 2.0.0
+---
+
+# Export-AzureRmLogAnalyticRequestRateByInterval
+
+## SYNOPSIS
+Export logs that show Api requests made by this subscription in the given time window to show throttling activities.
+
+## SYNTAX
+
+```
+Export-AzureRmLogAnalyticRequestRateByInterval [-FromTime] <DateTime> [-GroupByOperationName]
```

Comment:
Regenerate this once you reorder the parameters in cs file so that this syntax makes sense.

Reply:
I think the help generator has a bug.  The parameters have positional values except switch parameters, but the help generator does not generate Syntax correctly according to the positional values.

Reply:
We don't own the help generator, so if you would like to file a bug you can do so here: https://github.com/PowerShell/platyPS.  For now, the best way to solve this issue is order the parameters in the cs file in the order you want them shown in the syntax.